cmd/pessimism/main.go

```diff
@@ -20,36 +23,66 @@ const (
 
 // main ... Application driver
 func main() {
+	ctx := context.Background() // Create context
+	logger := logging.WithContext(ctx)
+
+	app := cli.NewApp()
+	app.Name = "pessimism"
+	app.Usage = "Pessimism Application"
+	app.Description = "A monitoring service that allows for " +
+		"Op-Stack and EVM compatible blockchains to be continuously assessed for real-time threats"
+	app.Action = RunPessimism
+	app.Commands = []cli.Command{
+		{
+			Name:        "doc",
+			Subcommands: doc.Subcommands,
+		},
+	}
```

Comment:
Shouldn't we also embed a command for running the application?

Reply:
We could, not sure we need to though. It'll run if you run the binary without any args.